R/build_site.R

```diff
@@ -34,25 +34,18 @@ update_site <- function(repo = "rstudio/sparklyr") {
   ## Copy the repo
   if(dir_exists("repos")) dir_delete("repos")
   copy_repo(repo)
-  
+
   ## Copy reference files
-  help_files <- dir_ls("content/reference", glob = "*.html")
+  help_files <- dir_ls("static/reference", glob = "*.html")
```

Comment:
Alternatively we can also embed the content in an iframe so that the left-hand-side nav bar built by Hugo will still be accessible while browsing sparklyr reference.

Iframe is usually frowned upon though.